core/src/script/CGXP/plugins/WFSPermalink.js

```diff
@@ -227,6 +236,10 @@ cgxp.plugins.WFSPermalink = Ext.extend(gxp.plugins.Tool, {
 
         var layerName = state.layer;
         delete state.layer;
+        if (state.show) {
+            this.showFeatures = false;
+            delete state.show;
```

Comment:
Because otherwise the plugin tries to call features using a condition on an attribute `show` (or `layer`), which doesn't exist in the feature's DB table, thus we have to delete it in order for it not to be used when querying features from the layer...